WcaOnRails/app/models/post.rb

```diff
@@ -34,7 +34,7 @@ def self.crash_course_post
   end
 
   def deletable
-    !is_crash_course_post?
+    !new_record? && !is_crash_course_post?
```

Comment:
Ah, good point! @jonatanklosko's seems technically more correct, in that it will not consider a destroyed post to be deletable, so I went with:

```ruby
  def deletable
    persisted? && !is_crash_course_post?
  end
```

I read this as "in order for it to be deletable, it has to be in the database, and it can't be the crash course!". @larspetrus, with your's, I feel like I need to do some slight extra mental gymnastics.